libsplinter/src/admin/store/diesel/mod.rs

```diff
@@ -252,3 +252,34 @@ impl AdminServiceStore for DieselAdminServiceStore<diesel::sqlite::SqliteConnect
         AdminServiceStoreOperations::new(&*self.connection_pool.get()?).list_services(circuit_id)
     }
 }
+
+#[cfg(all(test, feature = "sqlite"))]
+pub mod tests {
+    use crate::admin::store::diesel::migrations::run_sqlite_migrations;
+
+    use diesel::{
+        r2d2::{ConnectionManager, Pool},
+        sqlite::SqliteConnection,
+    };
+
+    #[test]
+    fn test_sqlite_migrations() {
+        create_connection_pool_and_migrate();
+    }
+
+    /// Creates a conneciton pool for an in-memory SQLite database with only a single connection
+    /// available. Each connection is backed by a different in-memory SQLite database, so limiting
+    /// the pool to a single connection insures that the same DB is used for all operations.
+    fn create_connection_pool_and_migrate() -> Pool<ConnectionManager<SqliteConnection>> {
```

Comment:
Some typos in the test comment here,  conneciton/connection, insures/ensures

Reply:
Fixed the typos